site/Calendar.php

```diff
@@ -0,0 +1,29 @@
+<html>
```

Comment:
A lot of this code at the top of the page should be in the `PrintHeader` function. The way to decide is to ask yourself if this exact html shows up anywhere else. If it does, it probably needs to go in a function.

This is true of functions in general. If you're writing the exact same code more than once, consider if a function would be better.

In this case, line 7 (the title of the page) isn't 100% exactly the same, but that can still go in the function. I'll leave the exact implementation as an exercise for you, but let me know if you're confused by that.

Reply:
I updated the header and added both the header and footer to init.php. I figured out the page titles as well. I think I may have messed up with branches though. I made all the edits on Header, which doesn't include the array stuff. If Header gets merged to Master, will those changes trickle down to Array?

Reply:
Sometimes this stuff can get a bit messy and have "conflicts" where github doesn't know what to do. In those cases, you need to manually resolve conflicts which is a pain in the ass, but not a big deal. We can worry about that if it happens later.